sqls/postgresql/create_openelis_db.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -eu
+
+function create_user_and_database() {
+	local database=$1
+	local user=$2
+	local password=$3
+	echo "  Creating 'OpenELIS' user and database..."
+	psql -v ON_ERROR_STOP=1 --username postgres postgres <<-EOSQL
+	    CREATE USER $user WITH UNENCRYPTED PASSWORD '$password';
+	    CREATE DATABASE $database;
+	    GRANT ALL PRIVILEGES ON DATABASE $user TO $database;
```

Comment:
The correct syntax is: `GRANT ALL PRIVILEGES ON DATABASE $database TO $user;`
Could you also update the other scripts with this change?